ui/src/components/field/QField.sass

```diff
@@ -190,11 +191,13 @@ $field-transition-label-right-up: .324s cubic-bezier(.4,0,.2,1)
     text-transform: inherit
     transform-origin: left top
     transition: transform $field-transition, max-width $field-transition-label-right-up
+    backface-visibility: hidden
 
   &--float .q-field__label
     max-width: 133%
     transform: translateY(-40%) scale(.75)
     transition: transform $field-transition, max-width $field-transition-label-right-down
+    backface-visibility: hidden
```

Comment:
Why is this needed, it is already set at line 194?
Am I missing something (like a quirk that makes this required)?

Reply:
I was sure backface visibility was broken again after transforming on Chrome(when getting `q-field--float` class). Yesterday I checked your code from 8dccbd4 and it worked without line 200. So I guess it is not really needed, sorry. Thank you for your review.

Reply:
No problem, I just wanted to check if it was required or not :)
Thank you